theano/sandbox/cuda/tests/test_fftconv.py

```diff
@@ -64,106 +64,6 @@ def test_full(self):
                       filters_shape=(2, 3, 3, 3),
                       border_mode='full', pad=True)
 
-    def test_opt_valid(self):
```

Comment:
I don't like the idea of just deleting the tests. If we remove it, we should remove the optimizations too and remove it from the doc. I'm not sure we should remove it... If someone could tell what they think, it would be great. @lamblin ?